src/lib.rs

```diff
@@ -240,6 +240,11 @@ pub mod auth {
                     // update state
                     JTI.replace(jti);
                     *cached_jwt = Some((jwt.to_string(), payload.clone()));
+                    log!(
```

Comment:
please notice that currently logs will not be produced for a cached (already validated JWT) -> https://github.com/neondatabase/pg_session_jwt/blob/072d8e57ac55e3eb741c91cb07e4281b0813eee0/src/lib.rs#L219

Reply:
Given that `validate_jwt` is called by both auth.jwt_session_init and auth.user_id, I assume both calls won't log?

This should be fine for `jwt_session_init`, but less so for `user_id` (since it means subsequent (e.g. via RLS) queries will not log the executor's sub.

Reply:
fixed in 1ae76de2251d